src/main/kotlin/com/openlattice/chronicle/services/timeusediary/TimeUseDiaryService.kt

```diff
@@ -154,4 +138,37 @@ class TimeUseDiaryService(
         preparedStatement.setObject(index, endDate)
         return preparedStatement.executeQuery()
     }
+
```

Comment:
@geekbeast As suggested, I added Dokka documentation to the refactored SQL. Am not 100% confident that this is the best way to refactor the sql with annotated parameters, so let me know if you had something different mind.